controllers/admin/AdminBackupController.php

```diff
@@ -1,318 +0,0 @@
-<?php
-/**
- * 2007-2018 PrestaShop
- *
- * NOTICE OF LICENSE
- *
- * This source file is subject to the Open Software License (OSL 3.0)
- * that is bundled with this package in the file LICENSE.txt.
- * It is also available through the world-wide-web at this URL:
- * https://opensource.org/licenses/OSL-3.0
- * If you did not receive a copy of the license and are unable to
- * obtain it through the world-wide-web, please send an email
- * to license@prestashop.com so we can send you a copy immediately.
- *
- * DISCLAIMER
- *
- * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
- * versions in the future. If you wish to customize PrestaShop for your
- * needs please refer to http://www.prestashop.com for more information.
- *
- * @author    PrestaShop SA <contact@prestashop.com>
- * @copyright 2007-2018 PrestaShop SA
- * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- * International Registered Trademark & Property of PrestaShop SA
- */
-
-@ini_set('max_execution_time', 0);
```

Comment:
what would be the best way to deal with this configuration in new pages?

Reply:
the best way is to use [streamed response](https://symfony.com/doc/current/components/http_foundation.html#streaming-a-response) instead of making the `max_execution_time` unlimited, which is a bad idea.

Reply:
i think it's not for downloading, but rather for creating backup, which may take some time. :/

Reply:
is it a blocking operation? do we need to use this backup directly after his creation?

Reply:
if i understand it correctly, yes, we need to use created backup. After backup creation, we show additional block with Backup's size and button to download it.